cucumber/authenticators_gcp/features/authn_gcp.feature

```diff
@@ -23,19 +23,22 @@ Feature: GCP Authenticator - Hosts can authenticate with GCP authenticator
     """
     And I am the super-user
     And I have host "test-app"
-    And I set GCP annotations to host "test-app"
     And I grant group "conjur/authn-gcp/apps" to host "test-app"
 
   Scenario: Hosts can authenticate with GCP authenticator and fetch secret
     Given I have a "variable" resource called "test-variable"
-    And I permit host "test-app" to "execute" it
     And I add the secret value "test-secret" to the resource "cucumber:variable:test-variable"
-    And I obtain a GCE identity token in "full" format and with audience claim value: "conjur%2Fcucumber%2Ftest-app"
+    And I set authn-gcp/service-account-id annotation to host "test-app"
+    And I set authn-gcp/service-account-email annotation to host "test-app"
+    And I set authn-gcp/project-id annotation to host "test-app"
+    And I set authn-gcp/instance-name annotation to host "test-app"
+    And I permit host "test-app" to "execute" it
```

Comment:
this should move upwards with the other variable steps

Reply:
these merges are killing me